### PR TITLE
GRW-1745 / feat / Add standalone video block

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes

First step for PDP product video
- Add a simple video block to start with
- Rename asset type to match Storyblok field name

## Justify why they are needed
Make it possible to add videos throughout the page. Will add more settings and functionality in upcoming PRs

https://user-images.githubusercontent.com/6661511/201108450-13d34b15-ba77-4ef4-912d-dc7eb47da592.mov

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #839** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/839" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/839?utm_source=stack-comment).

File: apps/store/src/blocks/VideoBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Video } from '@/components/Video/Video'
+import { SbBaseBlockProps, StoryblokAsset } from '@/services/storyblok/storyblok'
+
+type VideoBlockProps = SbBaseBlockProps<{
+  video: StoryblokAsset
+  poster?: StoryblokAsset
+}>
+
+export const VideoBlock = ({ blok }: VideoBlockProps) => {
+  console.log('test', blok.video)
```
Comment (alebedev) [inline apps/store/src/blocks/VideoBlock.tsx:10]: Forgotten debug log?

File: apps/store/src/components/Video/Video.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+})
+
+const StyledVideo = styled.video({
+  width: '100%',
+  height: 'auto',
+  backgroundSize: 'cover',
+  objectFit: 'cover',
+})
+
+export const Video = ({ sources, poster }: HeroVideoProps) => {
```
Comment (alebedev) [inline apps/store/src/components/Video/Video.tsx:27]: Can we reflect background use case in block name?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/Video/Video.tsx (hunk 2 above)
Comment (gustaveen) [reply apps/store/src/components/Video/Video.tsx:27]: Let's see if we should call it something else, the thought is to have this as a generic video block (if possible) where the editor can choose if it should be autoplay, have controls and such :)

**Pull request merged**